config/stamp/stamp.yml

```diff
@@ -12,16 +12,27 @@ XQUEUE_MYSQL_HOST: "10.0.0.16"
 EDXAPP_MYSQL_HOST: "10.0.0.16"
 EDXAPP_MEMCACHE: ["10.0.0.16:11211"]
 MONGO_BIND_IP: "0.0.0.0"
-FORUM_MONGO_HOSTS: ["%%MONGO_REPLICASET_NAME%%/10.0.0.11,10.0.0.12,10.0.0.13"]
+FORUM_MONGO_HOSTS:
+  - "10.0.0.13"
+  - "10.0.0.11"
```

Comment:
is there a reason why the order is 13, 11, 12 ?

also, you may get feedback about moving these ip addresses out to oxa-tools-config (files like bvt.sh), but i've been given that feedback before. i'll open a separate bug to track that work today. I think we should address it after eltonc is merged into master